app/models/katello/glue/candlepin/consumer.rb

```diff
@@ -88,6 +88,25 @@ def checkin(checkin_time)
         raise e
       end
 
+      def pool_quantities
+        quantity_table = {}
```

Comment:
Table is a confusing term to use here to me given it generally equates to a database table